manager/misc.go

```diff
@@ -55,14 +55,14 @@ type BundleMeta struct {
 //     |- <container1>.log
 //     |- ...
 // The bundle would be compressed to a zip file for download.
-func (m *VolumeManager) GenerateSupportBundle() (io.ReadCloser, string, int64, error) {
+func (m *VolumeManager) GenerateSupportBundle() (*SupportBundle, error) {
 	namespace, err := m.ds.GetLonghornNamespace()
 	if err != nil {
-		return nil, "", 0, errors.Wrapf(err, "cannot get longhorn namespace %v", err)
+		return nil, errors.Errorf("cannot get longhorn namespace %v", err)
```

Comment:
errors.Wrap(err, "xxx") is better here (and below). We won't lose the original error type.

Also, for Errorf, we normally use `fmt.Errorf`.